magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationTraverser.kt

```diff
@@ -0,0 +1,34 @@
+package com.wealthfront.magellan.navigation
+
+import com.wealthfront.magellan.core.Journey
+import com.wealthfront.magellan.core.Navigable
+
+class NavigationTraverser(private val root: Journey<*>) {
```

Comment:
Why don't we use `LifecycleOwner` here?

Reply:
This doesn't need to be a `LifecycleOwner `

Reply:
Yeah I have no idea what I was thinking haha